test/functional/apps/dashboard/dashboard_state.js

```diff
@@ -278,13 +280,15 @@ export default function ({ getService, getPageObjects }) {
           await retry.try(async () => {
             const pieSliceStyle = await pieChart.getPieSliceStyle('80,000');
             // The default green color that was stored with the visualization before any dashboard overrides.
-            expect(pieSliceStyle.indexOf('rgb(87, 193, 123)')).to.be.greaterThan(0);
+            expect(pieSliceStyle.indexOf(euiPaletteColorBlind()[1])).to.be.greaterThan(0);
```

Comment:
why is it [1] instead of [0] - does this test target the second pie slice?

Reply:
Sorry it has to be RGB format of colors and also I think it can be any color in the default color palette.